src/services/routes/org.routes.ts

```diff
@@ -25,27 +37,24 @@ export default function registerRoutes(
     },
 
     async handler(request, reply): Promise<void> {
-      const { name, awalaEndpoint } = request.body;
-
-      const isNameValid = isValidDomain(name, { allowUnicode: true });
-      if (!isNameValid) {
-        await reply.code(HTTP_STATUS_CODES.BAD_REQUEST).send({
-          type: ProblemType.MALFORMED_ORG_NAME,
-        });
-        return;
-      }
-
-      if (awalaEndpoint !== undefined && !isValidDomain(awalaEndpoint, { allowUnicode: true })) {
-        await reply.code(HTTP_STATUS_CODES.BAD_REQUEST).send({
-          type: ProblemType.MALFORMED_AWALA_ENDPOINT,
-        });
+      const result = await createOrg(request.body, {
+        logger: this.log,
+        dbConnection: this.mongoose,
+      });
+      if (result.didSucceed) {
+        await reply
+          .code(HTTP_STATUS_CODES.OK)
+          .header('Content-Type', 'application/json')
```

Comment:
Can you try dropping this? It shouldn't be needed -- plus, Fastify would automatically specify the charset, which is a good idea.